packages/synapse-react-client/src/components/styled/EvenlyDistributedWrappedContainer.tsx

```diff
@@ -0,0 +1,80 @@
+import { StyledComponent } from '@emotion/styled'
+import { Box, BoxProps, styled } from '@mui/material'
+import { chunk } from 'lodash-es'
+import { ReactNode } from 'react'
+
+const MAX_ITEMS_PER_ROW = 4
```

Comment:
This utility wrapper is specific to showing 4 items per row, because the 'evenly distributed' implementation wasn't designed for this to be parameterized.

It might be possible, but it would be more work without a known use case right now.